From 42ff5e80c99678818b090671020f442b4341a4cb Mon Sep 17 00:00:00 2001
From: Xinyu Chen <xinyu.chen@freescale.com>
Date: Thu, 15 Dec 2011 15:02:56 +0800
Subject: [PATCH 414/691] ENGR00170372 mx6q sabrelite: make TF card as default
 boot media

As we want TF to be default boot media.
Then SD slot can be used by WIFI dongle.

Signed-off-by: Xinyu Chen <xinyu.chen@freescale.com>
---
 include/configs/mx6q_sabrelite_android.h |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/include/configs/mx6q_sabrelite_android.h b/include/configs/mx6q_sabrelite_android.h
index 480101a..d32a26c 100644
--- a/include/configs/mx6q_sabrelite_android.h
+++ b/include/configs/mx6q_sabrelite_android.h
@@ -98,7 +98,7 @@
 	"setenv bootargs ${bootargs} init=/init root=/dev/mmcblk0p4 rootfs=ext4 rootwait"
 #define CONFIG_ANDROID_RECOVERY_BOOTCMD_MMC  \
 	"run bootargs_android_recovery; "	\
-	"mmc dev 0; "	\
+	"mmc dev 1; "	\
 	"mmc read ${loadaddr} 0x800 0x2000;bootm"
 #define CONFIG_ANDROID_RECOVERY_CMD_FILE "/recovery/command"
 
@@ -150,7 +150,7 @@
 		"bootfile=uImage\0"	\
 		"bootargs=console=ttymxc1 init=/init " \
 			"androidboot.console=ttymxc1 video=mxcfb0:bpp=16 fbmem=5M\0" \
-		"bootcmd_SD=mmc dev 0;"		\
+		"bootcmd_SD=mmc dev 1;"		\
 			"mmc read ${loadaddr} 0x800 0x2000;" \
 			"mmc read ${rd_loadaddr} 0x3000 0x300\0" \
 		"bootcmd=run bootcmd_SD; bootm ${loadaddr} ${rd_loadaddr}\0" \
-- 
1.7.7.4

